internal/pkg/deploy/env.go

```diff
@@ -13,7 +13,7 @@ const (
 	// LegacyEnvTemplateVersion is the version associated with the environment template before we started versioning.
 	LegacyEnvTemplateVersion = "v0.0.0"
 	// LatestEnvTemplateVersion is the latest version number available for environment templates.
-	LatestEnvTemplateVersion = "v1.4.0"
+	LatestEnvTemplateVersion = "v1.5.0"
```

Comment:
Just a reminder if https://github.com/aws/copilot-cli/pull/2413 is merged earlier we need to include changes in `v1.5.0`